fizzbuzz.js

```diff
@@ -0,0 +1,26 @@
+  const fizzbuzz = (number) => {
+    if ((number % 3 === 0) && (number % 5 === 0)) {
+        return 'fizzbuzz';
+    } else if (number % 3 === 0) {
+        return 'fizz';
+    } else if (number % 5 === 0) {
+        return 'buzz';
+    } else {
+        return number;
```

Comment:
Just `return number` at the end of the function instead of enclosing it in the `else` branch.